.env.example

```diff
@@ -4,4 +4,4 @@ STRIPE_SECRET_KEY=sk_test_1234
 STRIPE_WEBHOOK_SECRET=whsec_1234
 
 # Environment setup
-STATIC_DIR=../../client
\ No newline at end of file
+STATIC_DIR=../../client
```

Comment:
have we tested web still works?  Not sure this path is still right (should it be `../../client/web` or something?)

Reply:
uhhhh I thought I did but it does seem like it should be `/web` I'll double check

Reply:
I'm going to add a .cli.json today and test 👍

Reply:
Actually I'm going to add a .cli.json on a different branch after you merge this. We'll rename this repo to accept-ideal-bank-payment and that will affect the .cli.json.